lib/ui/home/loading/home_loading.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:shimmer/shimmer.dart';
+import 'package:survey_flutter_ic/ui/home/loading/home_loading_text.dart';
+
+class HomeLoading extends StatelessWidget {
+  const HomeLoading({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Shimmer.fromColors(
+      baseColor: Colors.white38,
+      highlightColor: Colors.white70,
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          _buildHeaderLoading(),
+          _buildBodyLoading(context),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildHeaderLoading() => Container(
+        padding: const EdgeInsets.only(
+          top: 10,
+          right: 20,
+          bottom: 10,
```

Comment:
Please use dimension value here for reuseable code
```suggestion
          top: Dimensions.paddingSmall,
          right: Dimensions.paddingMedium,
          bottom: Dimensions.paddingSmall,
```